README.md

```diff
@@ -50,6 +50,32 @@ You can also build just one of the binaries in this repo by passing a name to th
 $ make build BINARIES=amtool
 ```
 
+## Docker
+
+<!--- example of complete docker run command to run a container --->
+<!--- example of a docker-compose.yml service snippet --->
+
+### Ports
+
+* `9093` : main api port
+
+### Volumes
+
+* `/alertmanager`: storage directory
+
+These paths can also be mapped to volumes:
+* `/etc/alertmanager/alertmanager.yml`: default configuration file path
+
+### Environment variables
```

Comment:
I am not aware of any. Configuration is limited to the config file and the command line flags.

Reply:
Same for me. I'd remove this section.